store-server/store/apps/shop/models.py

```diff
@@ -161,6 +165,11 @@ class NewesBanner(models.Model):
     def __str__(self):
         return str(self.subject)
 
+    def save(self, *args, **kwargs):
```

Comment:
use `clean()` method for validation, try to avoid using `save()` method if it is not required